### PR TITLE
Add cpu hotplug support for ppc platform

### PR DESCRIPTION
**Pull request by maxujun**
ID: 1922000
Signed-off-by: Xujun Ma <xuma@redhat.com>

### CODE REVIEW

**Comment by maxujun** [comment]
@MiriamDeng Hi Could you help review?

**Comment by MiriamDeng** [comment]
Bug id should be on the first line and title should be similar with "cpu_hotplug:xxxxx"

**Comment by maxujun** [comment]
> 
> 
> Bug id should be on the first line and title&commit log should be similar with "cpu_hotplug:xxxxx"

Updated

File: qemu/tests/timedrift_check_when_hotplug_vcpu.py [hunk 1]
```diff
@@ -41,6 +42,7 @@ def run(test, params, env):
     query_times = int(params.get("query_times", "4"))
     query_internal = float(params.get("query_internal", "600"))
     drift_threshold = float(params.get("drift_threshold", "3"))
+    hotplug_cmd = params.get("cpu_hotplug_cmd", "")
```
Comment (MiriamDeng) [inline qemu/tests/timedrift_check_when_hotplug_vcpu.py:45]: Should we use  params.get("cpu_hotplug_cmd") ?

Comment (maxujun) [reply qemu/tests/timedrift_check_when_hotplug_vcpu.py]: Updated

**Comment by MiriamDeng** [comment]
It took a quite while,is it a product issue ? Thanks.
 (1/1) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.ppc64le.io-github-autotest-qemu.check_clock_offset.hotplug_vcpu: PASS (-15311.57 s)  - >

**Comment by maxujun** [comment]
> 
> 
> It took a quite while,is it expected in product ? Thanks.
> (1/1) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.ppc64le.io-github-autotest-qemu.check_clock_offset.hotplug_vcpu: PASS (-15311.57 s) - >

I have no this situation,Can you reproduce it?

**Comment by MiriamDeng** [comment]
LGTM
 (1/2) repeat1.Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.ppc64le.io-github-autotest-qemu.check_clock_offset.hotplug_vcpu: PASS (2543.87 s)

**Comment by maxujun** [comment]
@yanan-fu Hi,Could you help review this patch?

**Comment by PaulYuuu** [comment]
LGTM.
@YanhuiMa could you also help review this patch? Thanks in advance.

**Comment by YanhuiMa** [comment]
> LGTM.
> @YanhuiMa could you also help review this patch? Thanks in advance.

LGTM
(1/1) Host_RHEL.m8.u4.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.check_clock_offset.hotplug_vcpu.q35: PASS (111.22 s)

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**